### DISCUSSION TITLE
Add option to backup the system using timeshift before updating

### DISCUSSION THREAD

**Discussion by sanjyay** [prasanthrangan/hyprdots#1494] [Ideas]
![image](https://github.com/prasanthrangan/hyprdots/assets/102979855/58a9a835-8fc9-4e8b-a976-2899f1dd6ad3)
 i have added this code in the starting of systemupdate.sh. 

My intention is that i should be asked if i want to update the system and then the updating of the system should occur. 

But this option isn't showing up when i click the update button from waybar and I am not getting any errors too. 

I am very new to this . What am i doing wrong?

**Comment by kRHYME7** [comment]
Find the ` command=" ... " ` variable. then add your script there.
